http/vibe/http/auth/basic_auth.d

```diff
@@ -105,4 +105,12 @@ bool checkBasicAuth(scope HTTPServerRequest req, scope PasswordVerifyCallback pw
 	return false;
 }
 
+deprecated("It will be removed in 0.9. Use vibe.http.auth.basic_auth_client.addBasicAuth instead.")
+void addBasicAuth(scope HTTPRequest req, string user, string password)
```

Comment:
No need for a full fledged symbol, `deprecated alias` will do (and then you don't need to do the symbol rename in imported places).

Reply:
Wow, I didn't know about such an opportunity. Fixed.